db/DeliveryDB.go

```diff
@@ -0,0 +1,64 @@
+package db
+
+import (
+	"database/sql"
+	"log"
+)
+
+type DeliveryPostgres struct {
+	db *sql.DB
+}
+
+func NewDeliveryPostgres(db *sql.DB) *DeliveryPostgres {
+	return &DeliveryPostgres{db: db}
+}
+
+type Order struct {
+	IdDeliveryService uint16 `json:"delivery_service_id"`
+	Id                uint16 `json:"id"`
+	IdCourier         uint16 `json:"courier_id"`
+	DeliveryTime      string `json:"delivery_time"`
+	CustomerAddress   string `json:"customer_address"`
+	Status            string `json:"status"`
+	OrderDate         string `json:"order_date"`
+	RestaurantAddress string `json:"restaurant_address"`
+	Picked            bool   `json:"picked"`
+}
+
+func (r *DeliveryPostgres) GetActiveOrdersFromDB(Orders *[]Order) error {
+	db := ConnectDB()
+	defer db.Close()
+
+	insertValue := `Select * from delivery where status = 'ready to delivery'`
```

Comment:
better not to use `select *` when you need a specific order of fields, because some time later it could happen that multiple db migrations with alter table changed fields differently for different environments and now you couldn't rely on the order anymore